proImages/models.py

```diff
@@ -4,7 +4,7 @@
 class Images(models.Model):
     images_ID = models.BigAutoField(primary_key=True)
     # professional_ID = models.ForeignKey(Professional, on_delete=models.CASCADE)
```

Comment:
Either remove this, and open Issue once the pro model exist to add it 
You can replace the pro_id with String

Reply:
Replaced it with a dummy professional_id